Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -326,7 +326,11 @@ def each_resolved_path(action, paths)
             next
           end
 
-          yield path, Pathname.glob(resolved_path)
+          begin
+            yield path, Pathname.glob(resolved_path)
+          rescue
+            odie "Unable to remove some files. Please enable Full Disk Access for your teminal in settings."
```

Comment:
Instead of `in settings`, it would be best to be specific so users don’t have to hunt around: `under System Preferences → Security & Privacy → Privacy → Full Disk Access`.

Reply:
Yes, I agree that's definitely better. Thanks for the suggestion!